packages/editor/src/components/deprecated.js

```diff
@@ -66,7 +66,7 @@ function deprecateComponent( name, Wrapped, staticsToHoist = [] ) {
 		deprecated( 'wp.editor.' + name, {
 			since: '5.3',
 			alternative: 'wp.blockEditor.' + name,
-			version: '6.2',
+			version: '6.8',
```

Comment:
I've updated all the "refactor editor to block-editor" APIs that happened on WP 5.3 to 6.8. I kind of randomly picked that number because there's still some usage for these APIs but removing them at some point will have a somewhat good impact on potentially bundle size.

Reply:
I think it's almost 3kb for these.